src/components/components/club-full/club-full.js

```diff
@@ -42,17 +42,18 @@ function ClubFull(props) {
     const path = window.location.pathname.split("/");
     const id = (props.club || {}).id || path[path.indexOf("clubs") + 1];
     getClub(id).then(updateClub);
-  }, []);
+  }, [getClub, props.club, updateClub]);
```

Comment:
```suggestion
  }, []);
```

Reply:
I found out what's going on with this line.. It is being auto corrected on commit. That's why this code keeps coming back...

Reply:
Add: 
// eslint-disable-next-line react-hooks/exhaustive-deps
before this line